src/utils/request.ts

```diff
@@ -117,10 +129,56 @@ async function request(config: Config): Promise<OSDataHubResponse> {
     featureCount = output.results.length;
   }
 
-  logEndConditions(config);
+  logEndConditions(config, featureCount);
+
   if (typeof output === "undefined") {
     throw Error("There is no output at the end of request");
-  } else {
-    return output;
   }
+
+  return output;
+}
+
+async function requestNGD(config: Config): Promise<FeatureCollection> {
+  let endpoint: string;
+  let featureCount = 0;
+  let output: FeatureCollection | undefined = {
+    type: "FeatureCollection",
+    features: [],
+  };
```

Comment:
If we are making NGD-specific requests, should we try and put this stuff in a folder for ngd, rather than in a shared utils folder?

Reply:
Moved to ngd sub-directory